File: src/test/java/com/larryhsiao/clotho/source/ConstSourceTest.java
package com.larryhsiao.clotho.source;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

/**
 * Test for {@link ConstSource}
 */
public class ConstSourceTest {
    /**
     * Check output
     */
    @Test
    void normal() {
        Assertions.assertEquals(
            "TestString",
            new ConstSource<>("TestString").value()
        );
    }
}

File: src/test/java/com/larryhsiao/clotho/source/ConstSourceTest.kt
package com.larryhsiao.clotho.source

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class ConstSourceTest {
    @Test
    fun simple() {
        assertEquals(
            "TestString",
            ConstSource("TestString").value()
        )
    }
}